libraries/AP_RangeFinder/AP_RangeFinder.h

```diff
@@ -21,6 +21,10 @@
 #include <AP_MSP/msp.h>
 #include "AP_RangeFinder_Params.h"
 
+#ifndef AP_RANGEFINDER_ENABLE
+#define AP_RANGEFINDER_ENABLE 1
```

Comment:
This will save an additional 640B.
```suggestion
#define AP_RANGEFINDER_ENABLE 1
#define RANGEFINDER_MAX_INSTANCES 1
```

Reply:
Opps you are correct!! I always test these things by just setting the AP_LIBRARY_ENABLED 0 here. Which wont wont work for changing the rangefinders.

I'm guessing you have an easier way of testing the functionality of the #ifdef?